src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.hpp

```diff
@@ -367,6 +367,9 @@ class BrgemmEmitter : public jit_emitter {
     static std::set<std::vector<element::Type>> get_supported_precisions(const std::shared_ptr<ngraph::Node>& node = nullptr);
     size_t aux_gprs_count() const override;
 
+    static size_t get_in_leading_dim(const ov::Shape& shape, const std::vector<size_t>& layout);
```

Comment:
Minor: maybe we can pass `VectorDims` as parameters instead of Shape and vector<size_t>?